licenseheaders.py

```diff
@@ -71,7 +71,7 @@
     },
     "script": {
         "extensions": [".sh",".csh",".py",".pl"],
-        "keepFirst": re.compile(r'^#!'),
+        "keepFirst": re.compile(r'#!|# pylint| # -\*-'),

```

Comment:
not entirely sure that removing the "^" is a good idea. 

Having some sort of unittest for this would be nice to check the combinations of : 

* only hashbang
* only encoding header
* hashbash and pylint , etc.

Reply:
I believe this should match at the beginning of a line only as what is detected is literally the presence of the number 0x23 0x21 at the beginning of the file (I think some operating system versions may also allow the BOM first). 
```suggestion
        "keepFirst": re.compile(r'^#!|^# pylint|^# -\*-'),
```